config/Dockerfiles/ostree_image_compose/ostree-image-compose.sh

```diff
@@ -99,13 +99,22 @@ imagefactory --debug --imgdir $imgdir --timeout 3000 base_image /home/output/log
 imgname="fedora-atomic-$version-$release"
 qemu-img convert -c -p -O qcow2 $imgdir/*body /home/output/images/$imgname.qcow2
 
-# delete images over 3 days old
-find /home/output/images/ -type f -mtime +3 -exec rm {} \;
+pushd /home/output/images || exit 1
+latest=""
+if [ -e "latest-atomic.qcow2" ]; then
+    latest=$(readlink latest-atomic.qcow2)
+fi
+
+# delete images over 3 days old but don't delete what our latest link points to
+find . -type f -mtime +3 ! -name "$latest" -exec rm {} \;
```

Comment:
if these items are just files, should we add '-v' to rm to see what is it doing?